aws-proofs/vm-steps.sh

```diff
@@ -109,17 +110,44 @@ then
 else
   echo "Skipping image cache write"
 fi
+echo "::endgroup::"
 
-# bundle up logs for artifact upload outside the VM
+# Prepare artifacts for upload outside the VM
+mkdir -p ~/artifacts
+
+# Export C graph-lang for binary verification
+if [ -n "$INPUT_SIMPL_EXPORT" ]; then
+  echo "::group::C graph-lang export"
+  simpl_export_file="$l4v_dir/proof/asmrefine/export/$L4V_ARCH/CFunDump.txt"
+  if [ -f "$simpl_export_file" ]; then
+    echo Found CFunctions.txt
+    xz < "$simpl_export_file" > ~/artifacts/CFunctions.txt.xz
+  elif do_run_tests -L | grep -q SimplExportAndRefine; then
+    # SimplExportAndRefine was among the tests that should have run,
+    # which means we want a C graph export.
+    # However, we didn't get one, because the SimplExportAndRefine
+    # image was cached.
+    # So we force another export, but don't need to check refinement.
```

Comment:
How long does the export take on average?

Reply:
It's about 10 minutes.

Reply:
Cool, that makes sense. If it becomes an issue we could also try to rearrange the image structure so that we can do an export from a pre-built image in less time. But let's go with what we have for now.

Reply:
I think it's a good idea, but I would leave it for a later step. First get it going, then get it faster :-)